contributed/Derive_A_EBV_coefficients.ipynb

```diff
@@ -0,0 +1,165 @@
+    "    sed = photUtils.Sed()\n",
+    "    sed.readSED_flambda(sed_file)\n",
+    "    ax,bx = sed.setupCCM_ab()\n",
+    "    ccm_model = 3.1*ax+bx\n",
+    "    wl = sed.wavelen\n",
+    "    ccm_spline = scipy.interpolate.interp1d(wl,ccm_model,bounds_error=True)\n",
+    "    alam_over_ebv = 3.1*ax+bx\n",
+    "    filterset = ['u','g','r','i','z','y']\n",
+    "    for filter in filterset:\n",
+    "        if basepath is None:\n",
+    "            bp_file = os.path.join(os.environ['LSST_THROUGHPUTS_BASELINE'],'',f'total_{filter}.dat')\n",
+    "        else:\n",
+    "            bp_file = os.path.join(basepath,f'total_{filter}.dat')\n",
+    "        bandpass = photUtils.Bandpass()\n",
+    "        bandpass.readThroughput(bp_file)\n",
+    "        _,leff = bandpass.calcEffWavelen()\n",
+    "        lam_eff_list.append(leff)\n",
+    "        #print(f\"band {filter} leff1: {leff:.3f}\")\n",
+    "        alam = ccm_spline(leff)\n",
+    "        #print(f\"leff for band {filter}: {alam:.3f}\")\n",
+    "        alam_over_ebv_list.append(alam)\n",
+    "    return np.array(lam_eff_list),np.array(alam_over_ebv_list)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's do a quick check that we are getting the results that we expect.  For DC2 we should get the following for the effective wavelengths and A_lam/E(B-V) values:<br>\n",
+    "u 367.07 nm A_lambda/EBV = 4.812<br>\n",
+    "g 482.69 nm A_lambda/EBV = 3.643<br>\n",
+    "r 622.32 nm A_lambda/EBV = 2.699<br>\n",
+    "i 754.60 nm A_lambda/EBV = 2.063<br>\n",
+    "z 869.01 nm A_lambda/EBV = 1.578<br>\n",
+    "y 971.03 nm A_lambda/EBV = 1.313<br>\n",
+    "If these values do not match those in the next cell, check that the Baseline filter definitions have not changed!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "filterlist = ['u','g','r','i','z','y']\n",
+    "leff_list,alamebv_list = compute_alambda_over_ebv(filterlist)\n",
+    "for filt,leff, alamebv in zip(filterlist,leff_list,alamebv_list):\n",
+    "    print(f\"filter {filt} lam_eff: {leff:.2f}nm   alam/E(B-V): {alamebv:.3f}\")\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "These numbers do *not* match, there are very slight differences, as the filters were updated in mid 2019.  The filter curves used for DC2 are released as v1.4 cosmoDC2:\n",
+    "https://github.com/lsst/throughputs/releases/tag/1.4.  For convenience, we have saved a copy of these filters in the data/dc2_throughputs subdirectory.  So, we can calculate the lambda_eff and A_lambda/E(B-V) for these filters:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "filterlist = ['u','g','r','i','z','y']\n",
+    "leff_list,alamebv_list = compute_alambda_over_ebv(filterlist,basepath=\"data/dc2_throughputs\")\n",
```

Comment:
Need to check in this file `data/dc2_throughputs`

Reply:
I added the throughput files to the repo